src/Core/tests/DeviceTests/Stubs/ElementHandlerStub.cs

```diff
@@ -12,5 +12,7 @@ public ElementHandlerStub() : base(ElementHandler.ElementMapper)
 		}
 
 		protected override object CreatePlatformElement() => new Object();
+
+		public override void SetVirtualView(IElement view) => base.SetVirtualView(view);
```

Comment:
Yes, there is an exception propagation test https://github.com/dotnet/maui/blob/8ba0551fd0e38ac8c42012c9dcf74bf0dc95356b/src/Core/tests/UnitTests/HandlerTests.cs#L24-L77
that tests whether a view/element can throw an exception for another view/element.

I originally moved all `HandlerTests` to `HostBuilderHandlerTests` because I didn't see why we really needed the separation, both were in the same testing namespace anyways and both used HostBuilding (MauiApp.CreateBuilder().Build) 

Then I figured that the test didn't fit in either, as it wasn't testing the HostBuilder's ability to register and retrieve services, rather it would be an ElementTest, so now its there as `ElementToHandlerPropagatesThrownException`.

The reason `ElementToHandlerPropagatesThrownException` needs that declared `override void SetVirtualView` from what I'm guessing because I don't fully understand the intent of the original test, was to give it some easy access method to throw an exception in the nested view/element after grabbing the view/element's handler in the `ToHandler` extensions method https://github.com/dotnet/maui/blob/8ba0551fd0e38ac8c42012c9dcf74bf0dc95356b/src/Core/src/Platform/ElementExtensions.cs#L81-L99. So once the view/element (`ViewWithExceptionThrowingChildStub`) grabbed the nested child view/element which throws the exception (`ExceptionThrowingViewStub`) we just check that the exception propagates through. Originally it was using `CreatePlatformView` to throw the exception but I figured if we just wanted to propagate an exception being thrown, using `SetVirtualView` would achieve the same results. But moving from UnitTests to DeviceTests meant that different stubs were available `ViewStubHandler` vs `ElementStubHandler`, but I couldn't override `SetVirtualView` without adding this to the `ElementHandlerStub`. Maybe I was missing something simple?

So the propagation pattern now looks like
`ViewWithExceptionThrowingChildStub().ToHandler` -> `GetHandler` -> `ViewWithExceptionThrowingChildHandler.SetVirtualView` -> `ExceptionThrowingViewStub().ToHandler` -> `GetHandler` + `ExceptionThrowingViewHandler.SetVirtualView` -> throw `HandlerPropagatesException`.
Compared to the original test that had
`ViewStub().ToHandler` -> `GetHandler` -> `ViewHandlerWithChildException.SetVirtualView` -> ExceptionThrowingStub().ToHandler` -> `GetHandler` -> `ExceptionThrowingViewHandler.SetVirtualView` -> `ExceptionThrowingViewHandler.CreatePlatformView` -> throw `PlatformViewStubCreatePlatformViewException`

@PureWeen  do you happen to know if I'm on the right track for the original intent of the test?